gtfs_rt_validator_api.py

```diff
@@ -108,14 +147,25 @@ def validate_gcs_bucket(
         out_dir: a directory to store fetched files and results in.
         results_bucket: a bucket path to copy results to.
         verbose: whether to print helpful messages along the way.
+        aggregate_counts: tbd
 
     Note that if out_dir is unspecified, the validation occurs in a temporary directory.
-        
-    """
 
-    import gcsfs
+    """
+    # TODO: get python 3.9
+    clear_threadlocal()
+    bind_threadlocal(
+        idx=idx,
+        gtfs_schedule_path=gtfs_schedule_path,
+        gtfs_rt_glob_path=gtfs_rt_glob_path,
+        out_dir=out_dir,
+        result_bucket=results_bucket,
+    )
+    logger.info("entering validate_gcs_bucket")
```

Comment:
Is this maybe more of a debug-level log? Not sure what norms we want to have around that kind of thing (just seems a bit nitty-gritty for info, if we print info level by default?)

Reply:
Oh yeah that's fair; I always just default to `info()` but I should definitely be better about that; fixing

Reply:
looks like it's still `info` -- NBD but just checking if you had anything else you wanted to commit

Reply:
oops missed that, fixing

Reply:
this one does say debug now